charter.md

```diff
@@ -64,19 +64,19 @@ The current platform includes packages under a variety of licenses, including no
 
 ### Several tiers
 
-If it turns out that it would help providing timely releases or solving the licensing questions raised above, we might decide to provide several tiers of the Coq platform. For instance, we could provide both a core Coq platform that would be released only weeks after a Coq release and provide a basic set of Coq packages with strict inclusion criteria, and an extended Coq platform that would be released months after and provide a larger set of packages with more lax inclusion criteria. If we were to provide several tiers, the naming scheme for those is still to be determined.
+If it turns out that it would help providing timely releases or solving the licensing questions raised above, we might decide to provide several tiers of the Coq Platform. For instance, we could provide both a core Coq Platform that would be released only weeks after a Coq release and provide a basic set of Coq packages with strict inclusion criteria, and an extended Coq Platform that would be released months after and provide a larger set of packages with more lax inclusion criteria. If we were to provide several tiers, the naming scheme for those is still to be determined.
```

Comment:
Maybe this could be updated to say that we do plan a "core" and an "extended" selection, and that non-open-source packages have a specific selection mechanism.

Reply:
The charter (and also the system specific readme files) are still todo - I just unified the writing of "Coq Platform" here. But I keep a note.